bin/aurcheck

```diff
@@ -76,6 +76,19 @@ namever_aur() {
     xargs -r aursearch -ir | jq -er '.results[] | [.Name, .PackageBase, .Version] | @tsv'
 }
 
+namever_vcs() {
+    for pkg in "$@"; do
+        base=$(aursearch -ir "$pkg" | jq -er '.results[] | .PackageBase')
```

Comment:
This is needed for split packages, unless there is an easier way to get its folder name in `~/.cache`?